src/lib/types.ts

```diff
@@ -398,9 +446,20 @@ export enum SignalRequestSignalEnum {
   SIGUSR1 = 'SIGUSR1',
 }
 
+/**
+ * * no - Never try to restart a Machine automatically when its main process exits, whether that’s on purpose or on a crash.
+ * * always - Always restart a Machine automatically and never let it enter a stopped state, even when the main process exits cleanly.
+ * * on-failure - Try up to MaxRetries times to automatically restart the Machine if it exits with a non-zero exit code. Default when no explicit policy is set, and for Machines with schedules.
+ */
+export enum ApiMachineRestartPolicyEnum {
+  No = 'no',
+  Always = 'always',
+  OnFailure = 'on-failure',
+}
+
 export interface AppsListParams {
   /** The org slug, or 'personal', to filter apps */
-  org_slug?: string
```

Comment:
Also a breaking change. Previously this was optional because it defaults to `personal`. Did something change on fly api side which motivated this update?